src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.java

```diff
@@ -281,14 +282,25 @@ private VendorAddress assignMethodOfPoTransmission(VendorAddress poAddress, Paym
         }
         else if (StringUtils.equalsIgnoreCase(pmwVendor.getPoTransmissionMethod(), PaymentWorksConstants.PaymentWorksMethodOfPoTransmission.FAX.getPmwPoTransmissionMethodAsText())) {
             poAddressExtension.setPurchaseOrderTransmissionMethodCode(PaymentWorksConstants.PaymentWorksMethodOfPoTransmission.FAX.getKfsPoTransmissionMethodCode());
-            poAddress.setVendorFaxNumber(pmwVendor.getPoFaxNumber());
+            setVendorFaxNumberValue(poAddress, pmwVendor.getPoFaxNumber());           
         }
         else if (StringUtils.equalsIgnoreCase(pmwVendor.getPoTransmissionMethod(), PaymentWorksConstants.PaymentWorksMethodOfPoTransmission.US_MAIL.getPmwPoTransmissionMethodAsText())) {
             poAddressExtension.setPurchaseOrderTransmissionMethodCode(PaymentWorksConstants.PaymentWorksMethodOfPoTransmission.US_MAIL.getKfsPoTransmissionMethodCode());
         }
         poAddress.setExtension(poAddressExtension);
         return poAddress;
     }
+    
+    private void setVendorFaxNumberValue(VendorAddress vendorAddress, String vendorFaxNumber) {
+        try {
+            Class type = ObjectUtils.easyGetPropertyType(vendorAddress, VendorPropertyConstants.VENDOR_FAX_NUMBER);
+            ObjectUtils.setObjectProperty(vendorAddress, VendorPropertyConstants.VENDOR_FAX_NUMBER, type, vendorFaxNumber);
+        } catch (FormatException | IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
+            vendorAddress.setVendorFaxNumber(null);
+            LOG.error("setVendorFaxNumberValue: Vendor Fax Number cannot be set due to exception: " + e);
+            e.printStackTrace();
```

Comment:
It would be better to use Log4j to print the stack trace, rather than using the exception's printStackTrace() method.

Reply:
Thank you, I am updating this.